src/main/java/frc/robot/Constants.java

```diff
@@ -42,16 +42,16 @@ public final Boolean isEnabled() {
   public enum AimModes {
     AUTO,
     // TODO: Plug real values in for these aimModes
-    LOW(0.0, 0.0) {
+    LOW(3200.0, 0.0) {
       @Override
       public AimModes previous() {
         return values()[values().length - 1];
       }
     },
-    EJECT(0.0, 0.0),
+    EJECT(2500.0, 0.0),
```

Comment:
do we want to have this even lower RPM?  Wondering how quickly we can get sped up and let the opposing ball loose

Reply:
All RPMs are subject to change as testing progresses